### DISCUSSION TITLE
Share Your Favorite Gadgets! あなたの好きなガジェットを教えてください！

### DISCUSSION THREAD

**Discussion by ShunHattori** [NAIST-SE/AttractingContributors#1148] [2024 Attraction Contributions]
私達は情報系の学生なので，デバイスに触れる時間がとても長いと思います．
そんな時間をより楽しく，生産的にするあなたのお気に入りのガジェットをぜひ共有してください！

As computer science students, we spend a lot of time with our devices. 
Please share your favorite gadgets that make this time more enjoyable and productive!

以下のレポジトリをクローンし，READMEにあなたのお気に入りのガジェットを追加してください．
Please clone and edit the README file and make a PR.

My Repo Link : [https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)

**Comment by Drobotsan** [comment]
Hi, I send a pull request

**Reply by ShunHattori** [reply]
Thank you for your PR!

**Comment by kozmr** [comment]
Hi, I contributed to your project.
Could you contribute to my project?

https://github.com/kozmr/AttractingContributors-YouTube
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1113

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git
